.github/workflows/bench.yml

```diff
@@ -0,0 +1,30 @@
+name: Benchmark
+#  Benchmark workflow runs benchmarks (now, only for CosmWasmAPI)
+#  This workflow is run on pushes to develop & every Pull Requests where a .go, .mod, .sum have been changed
+env:
+  GOPRIVATE: "github.com/line/*"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - dynamic_link
+jobs:
+  benchmark:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2.1.4
+        with:
+          go-version: 1.15
```

Comment:
v1.18 is right?

Reply:
This (1.15) is following the `test.yml` in dynamic_link branch.